pyproject.toml

```
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "example"
+version = "0.1.0"
+requires-python = ">=3.10"
+
+[project.dependencies]
+numpy = "^1.21.0"
+matplotlib = "^3.4.0"
+
```

Suggesting packages that will likely be used for data access from websites and establishing connection to HTTP pages.
```suggestion
"beautifulsoup4>=4.12.0"
"requests>=2.30.0"
```